### DISCUSSION TITLE
Can `quarto publish gh-pages` use remote source other than `origin`

### DISCUSSION THREAD

**Discussion by rgaiacs** [quarto-dev/quarto-cli#5993] [Feature Requests]
### Description

I have a Git remote source named `quarto`:

```
$ git remote -v
quarto  git@github.com:gesiscss/orc2.git (fetch)
quarto  git@github.com:gesiscss/orc2.git (push)
```

Executing `quarto publish gh-pages` produces the follow prompt

```
? Publish site to undefined using gh-pages? (Y/n) › 
```

Will be great if the prompt get information of other remote sources.

**Comment by cderv** [comment]
Indeed, we should try improve this. I'll transform to an issue to track better this issue.

**Reply by mcanouil** [reply]
~~I did not check, but I don't think "origin" is hardcoded.
I believe it uses the default remote set in your local clone.~~

It is hardcoded:
https://github.com/quarto-dev/quarto-cli/blob/41501dd86dee2103cb5d88954303feed32f1d3b0/src/core/github.ts#L40

**Reply by rgaiacs** [reply]
Thanks for the issue (https://github.com/quarto-dev/quarto-cli/issues/5997). I'm marking as close.
